openff/bespokefit/cli/executor/submit.py

```diff
@@ -119,6 +131,21 @@ def _to_input_schema(
             workflow_factory.initial_force_field = force_field_path
         if len(target_torsion_smirks) > 0:
             workflow_factory.target_torsion_smirks = [*target_torsion_smirks]
+        if default_qc_spec is not None:
+
+            program, method, basis = default_qc_spec
+
+            if basis.lower() == "none":
+                basis = None
+
+            workflow_factory.default_qc_specs = [
+                QCSpec(
```

Comment:
if the user enters an invalid combination of program, method and basis the spec will raise an `QCSpecificationError ` [here](https://github.com/openforcefield/openff-qcsubmit/blob/439b117a79e554b4595d5871a6eeca87f123cbd3/openff/qcsubmit/common_structures.py#L496) which should also be caught and displayed as it will give hints on valid combinations. We probably also want a test for this.